doc/modules/classes.rst

```diff
@@ -1350,6 +1350,7 @@ Model validation
 
    neighbors.kneighbors_graph
    neighbors.radius_neighbors_graph
+   neighbors.sort_by_row_values
```

Comment:
> Happy to change to another name if needed. (sort_by_data, sort_graph_by_row_values, sort_csr_by_data, sort_csr_by_row_values, ... ?)

As a public API, I like `sort_graph_by_row_values`. It fits the naming of the other `neighbors` functions (`kneighbors_graph` and `radius_neighbors_graph`).